core/templates/dev/head/base.html

```diff
@@ -51,7 +51,11 @@
     <meta property="og:image" content="{{DOMAIN_URL}}/images/logo/288x288_logo_mint.png">
 
     <link rel="apple-touch-icon" href="/images/logo/favicon.png">
-    <title itemprop="name" ng-bind="pageTitle">{% block subtitle %}{% endblock %} - {% block maintitle %}Oppia{% endblock %}</title>
+    <!-- The title is binded to the rootScope and it will be replaced for its
+    translation when the page is loading. This is the only way to translate the
+    page title because the head of the file is outside the scope of any other
+    controller. -->
```

Comment:
I think we should explain here that the maintitle can be either an i18n code or a string, and that it will only be translated if it's an i18n code. I think it would also be nice to test that the behaviour is correct in both cases (i.e. if it's an i18n code, it gets translated, and if it's a string, there's no error raised).